ros2_ws/src/warehouse_robot/urdf/robot_parts/omni_wheel.xacro

```diff
@@ -4,7 +4,7 @@
 
     <xacro:macro name="omni_wheel" params="prefix pos_X pos_Y pos_Z pos_r pos_p pos_y">
         <joint name="joint_${prefix}_wheel" type="fixed">
-            <parent link="base_link" />
+            <parent link="carcaca" />
```

Comment:
I'm confused by this because it looks like we renamed the carcaca link to base_link, but does the carcaca link still exist?

Reply:
other way around.  base_link is not caracaca link, and looks like it was changed in the carcaca.xacro file too.

Reply:
So base_link is the root link of the robot and nothing else. carcaca is basically the chassis with all the geometry and inertia parameters attached to it. Both of them still exist.

In the previous revision I did rename carcaca to base_link though, which is probably what you're thinking of.